tla-io/src/test/scala/at/forsyte/apalache/tla/imp/TestSanyImporterStandardModules.scala

```diff
@@ -479,7 +479,7 @@ class TestSanyImporterStandardModules extends SanyImporterTestBase {
     expectSourceInfoInDefs(root)
     // This number may change when a new version of Bags.tla is shipped in tla2tools.jar.
     // The declarations include the declarations by __rewire_tlc_in_apalache.tla and Bags.tla.
-    assert(16 == root.declarations.size)
+    assert(14 == root.declarations.size)
```

Comment:
This was already merged, but why did this number drop again?
Is the comment above it outdated?